package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dedalusdiit/cornerstone-tools",
-  "version": "6.0.8-ded3",
+  "version": "6.0.8-ded7",
```

Comment:
Please note @p1p3 comment on what we can use here
```suggestion
  "version": "6.0.8-ded4-rc1",
```

Reply:
oh this is great, was wondering about this!!